src/server/csp.ts

```diff
@@ -67,7 +67,7 @@ const assetHashes = (assets: string[]): string =>
 
 const buildCsp = ({ styles, scripts }: Assets, twitter: boolean): string => `
     default-src 'self';
-    style-src 'self' 'unsafe-hashes' ${assetHashes(styles)} https://interactive.guim.co.uk ${twitter ? 'https://platform.twitter.com' : ''};
+    style-src https: 'unsafe-inline';
```

Comment:
This is something I really want to avoid if possible, it undoes a lot of the work of the CSP. How broken does it look without these `style` attributes?

Reply:
Unfortunately not great on Safari

<img width="630" alt="93602976-4c264b00-f9bb-11ea-9c4d-6fb760e06721" src="https://user-images.githubusercontent.com/11618797/93883094-029a7080-fcd9-11ea-8230-38e514b23b14.png">

Reply:
We could write a css-in-js like library to abstract these inline styles into `style` elements and generate unique classnames... 😅

Reply:
I don't know if you're serious but I did actually think about it 🙂. Decided that a) it might be too complicated and brittle, and b) we control atoms, if we're going to be doing a bunch of work I'd prefer to do it at the other end.